src/core.c/CompUnit/PrecompilationRepository.pm6

```diff
@@ -251,11 +259,10 @@ class CompUnit::PrecompilationRepository::Default does CompUnit::PrecompilationR
                 note "\n    load    $path.relative()";
                 $*ERR.flush;
             }
-            self.store.unlock;
+            #self.store.unlock;
```

Comment:
commented code
Shouldn't `$bcl` be unlocked and closed here?

Reply:
Would that be better left to a LEAVE block?